clusterDeployer.py

```diff
@@ -58,51 +61,65 @@ def setup_vm(h: host.LocalHost, virsh_pool: VirshPool, cfg: dict, iso_path: str)
     RAM_MB = 32784
     DISK_GB = 48
     CPU_CORE = 8
-    network = "default"
-
-    cmd = f"""
-    virt-install
-        --connect qemu:///system
-        -n {name}
-        -r {RAM_MB}
-        --vcpus {CPU_CORE}
-        --os-variant={OS_VARIANT}
-        --import
-        --network=network:{network},mac={mac}
-        --events on_reboot=restart
-        --cdrom {iso_path}
-        --disk pool={virsh_pool.name()},size={DISK_GB},sparse=false
-        --wait=-1
-    """
+
+    if hostname == "localhost":
+        network = "network=default"
+    else:
+        network = "bridge=virbr0"
+
+    cmd = f""" virt-install \
```

Comment:
do we need `\`?

Reply:
I think so. When using it through ssh on remote systems, the missing \ causes ssh to see virt-install command w/o any parameter, and all parameters as separate commands..